activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/error/attributes/ErrorAttributesModelValidationErrorsBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.modeling.rest.controller.error.attributes;
+
+import org.activiti.cloud.common.error.attributes.ErrorAttributesCustomizer;
+import org.activiti.cloud.modeling.api.ModelValidationError;
+import org.activiti.cloud.modeling.core.error.SemanticModelValidationException;
+import org.springframework.validation.ObjectError;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ErrorAttributesModelValidationErrorsBuilder implements ErrorAttributesCustomizer {
+
+    public static final String ERRORS = "errors";
+
+    @Override
+    public Map<String, Object> customize(Map<String, Object> errorAttributes, Throwable error) {
+
+        Stream<ModelValidationError> bindingErrors = Optional.ofNullable((List<ObjectError>) errorAttributes.get(ERRORS))
+            .map(this::transformBindingErrors)
+            .orElse(Stream.empty());
+        Stream<ModelValidationError> semanticErrors = resolveSemanticErrors(error,
+            errorAttributes);
+        Stream<ModelValidationError> modelValidationErrorStream = Stream.concat(bindingErrors,
+            semanticErrors);
+        List<ModelValidationError> collectedErrors = modelValidationErrorStream.collect(Collectors.toList());
+        if (!collectedErrors.isEmpty()) {
+            errorAttributes.put(ERRORS,
+                collectedErrors);
+        }
+
+        return errorAttributes;
+    }
+
+    private Stream<ModelValidationError> resolveSemanticErrors(Throwable error,
+                                                               Map<String, Object> errorAttributes) {
```

Comment:
The parameter `errorAttributes` is never used in this method, so you can remove it.

Reply:
Done in [56a57f4](https://github.com/Activiti/activiti-cloud/pull/905/commits/56a57f4c8040e95f9c1eb6a2a49979d0cdde2d16)